AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1936,6 +1936,7 @@ warezcrack.net#$#.onp-sl-content > [data-lock-id^="onpLock"] { display: block!im
 windowstechblog.nl#%#AG_onLoad(function() { window.bizpanda = {}; });
 windowstechblog.nl#$#.onp-sl-blur-area[style="filter: blur(5px);"] { filter: none!important; }
 gigadownloads.co,techsini.com##.onp-sl-overlap-box
+animaunt.tv##.icon-bar > .fl-fl
```

Comment:
what is it? Cannot find in dev tools

Reply:
![image](https://user-images.githubusercontent.com/15062885/61857901-addd4900-aecd-11e9-9091-5b03f73ed24b.png)